### ISSUE TITLE
Shortcut for Calc is the same as a shortcut for zoom the browser page 

### ISSUE THREAD

**Issue by abaranowska1** [benetech/MathShare#1147]
**Describe the bug**
Shortcut for Calc is the same as a shortcut for zoom the browser page ( Ctrl+ )

**To Reproduce**
1. Go to Problem
2. Mark some of the input ready to calc ( for example some adding or subtraction)  and click Ctrl +

**Actual result**
The calc is done, but the page also has been zoomed
**Expected behavior**
The calc is done, but without zooming the page

**Screenshots**
![Peek 2020-04-22 13-05](https://user-images.githubusercontent.com/61964176/79975772-cc439580-849b-11ea-95b2-076d54ddf176.gif)

**Desktop:**

- OS: [Ubuntu 16.04]
- Browser [Chrome, Firefox]
- Version [Version 81.0.4044.92 (Official Build) (64-bit); Version 75.0 (64-bit)]

**Comment by alexrcabral** [comment]
Keyboard shortcut for Calc should be changed to Control-Alt-Enter (PC) / Option-Command-Return (Mac). 

These appear to not conflict with OS-level keyboard shortcuts, VoiceOver, JAWS, NVDA, or existing Mathshare shortcuts, though @jscholes an extra once-over would be helpful.

**Comment by sinabahram** [comment]
That’s a pretty tough keystroke for lots of folks. Can we do shift+enter instead?

**Comment by alexrcabral** [comment]
Shift-enter is an existing shortcut in the math editor for adding a step. Are there other easier keystrokes that don't conflict with existing ones?

**Comment by sinabahram** [comment]
Where can all keyboard shortcuts for this application be found in one place? That's required before I can respond further to this topic.

**Comment by alexrcabral** [comment]
Update: Keyboard Shortcuts 

Keyboard Shortcuts:
https://github.com/benetech/MathShare/wiki/Keyboard-Shortcuts

Math Editor Behavior
https://github.com/benetech/MathShare/wiki/Math-Editor-Behavior

Updates:
Two separate pages — Keyboard Shortcuts and Math Editor Behavior
Key commands separated out by Windows/Linux shortcuts vs Mac shortcuts
Key commands translated appropriately (e.g. Delete on Windows as Fn-Delete on Mac, Option as Alt, etc)
Commands ordered appropriately based on OS recommendation
Where the unicode symbol is necessary for visual recognition, it’s now in a “Symbol” column next to the first column containing textual title of the Action/Operator/Notation
Tested all Mathshare commands and relevant native MathLive commands on Mac and Windows, removed deprecated commands, fixed errors and typos

**Comment by alexrcabral** [comment]
Update: Keyboard Shortcuts and Math Editor Behavior have been updated and put into two separate Github wiki pages: [Keyboard Shortcuts](url) & [Math Editor Behavior](https://github.com/benetech/MathShare/wiki/Math-Editor-Behavior.
)

I've tested all Mathshare commands and relevant native MathLive commands on Mac and Windows and with NVDA and Voiceover, removed deprecated commands, and fixed errors and typos. 

@bognasoldev89 The keyboard shortcuts in particular should be added to test cases.

**Comment by alexrcabral** [comment]
@sinabahram For calculate, would Alt+Equals (PC) / Option+Equals (Mac) be a better keystroke?

**Comment by sinabahram** [comment]
Alex, I would test that with users, but it does seem quite possible, yes, especially since it maps thematically with the operation and mimics a calculator interface.

**Comment by alexrcabral** [comment]
Solid, thanks Sina!

**Comment by alexrcabral** [comment]
Updated scoping: 
Keyboard shortcut for Calc should be changed to Control-Alt-Enter (PC) / Option-Command-Return (Mac), and the tool-tip should be updated to reflect this. 

When added, I will update the Github Keyboard Shortcuts page, and @benetechMatt will updated any external documentation (e.g. Intercom) referencing keyboard shortcuts. 

I've also added this as a topic for usability tests with students.

**Comment by bognasoldev89** [comment]
> @bognasoldev89 The keyboard shortcuts in particular should be added to test cases.

It sure will.

**Comment by bognasoldev89** [comment]
**Tested with success**

**Environment**
- QA env
- Browsers
-- Chrome v83.0.4103.116
-- Firefox v78.0.2

**Verified cases**
- Trying keyboard shortcuts, especially the one for Calc

**Summary**
The new shortcut for Calc works just fine and it's not difficult to reach it on the keyboard. The tool-tip reflects changes.
Test cases for keyboard shortcuts will be added to the test plan and will be executed starting with the next regression.

**Attachment**
![ms 1147](https://user-images.githubusercontent.com/51324797/87161366-b5575700-c2c4-11ea-8c8e-7d036283f8e6.gif)

**Comment by alexrcabral** [comment]
@bognasoldev89 Unfortunately, the tooltip does not accurately reflect changes. While the shortcut itself is correct, the tooltip is incorrect on Mac. This could be very confusing to students who are trying to use the shortcuts on a Mac, and don't see or know of any "Alt" or "Enter" key on their keyboard. 

@rupeshparab Please update to the aforementioned scope: 

> Keyboard shortcut for Calc should be changed to Control-Alt-Enter (PC) / **Option-Command-Return (Mac)**, and the tool-tip should be updated to reflect this.

I've also noticed a bug in some of the tooltips (also present in the live site), and have created #1293 to address this.

**Comment by rupeshparab** [comment]
@alexrcabral 
I have pushed a fix for this, now the tooltip will be displayed like this:
![image](https://user-images.githubusercontent.com/6897505/87810158-288b3b00-c87a-11ea-8174-1ef10c90bba3.png)

**Comment by alexrcabral** [comment]
Thanks @rupeshparab, looks good

**Comment by bognasoldev89** [comment]
Indeed, it's changed. Thanks for your catch @alexrcabral.
